### PR TITLE
Add support for PHP 8.0 and PHP 8.1

### PR DESCRIPTION
**Pull request by oskosk**
Adds these versions to the `available_php_versions()` array

### CODE REVIEW

**Comment by bluefuton** [comment]
@oskosk is there anything I can do to help out with this one? We're just releasing some PHP 8.1+ updates to Akismet and I'd love to be able to test them on JN.

We could also potentially drop PHP 5.4 from the list as that no longer successfully launches a WP site.

File: lib/settings-stuff.php [hunk 1]
```diff
@@ -20,6 +20,8 @@
  */
 function available_php_versions() {
 	return array(
+		'8.1' => __( '8.1', 'jurassic-ninja' ),
```
Comment (bluefuton) [inline lib/settings-stuff.php:23]: It'd be great to add 8.2 as well 👍

Comment (oskosk) [reply lib/settings-stuff.php:24]: Addressed in 6671beace23d5fb49e3004e6fedb6881156e58ab

Comment (bluefuton) [reply lib/settings-stuff.php:24]: Thanks @oskosk 🙌

**Pull request merged**